src/select/Input.js

```diff
@@ -16,6 +16,7 @@ const Input = ({
       id={id}
       aria-label={ariaProps['aria-label'] || undefined}
       aria-labelledby={ariaProps['aria-labelledby'] || undefined}
+      aria-multiline={ariaProps['aria-multiline'] || undefined}
```

Comment:
Why this addition?  I don't see where it would ever be set to true.

Reply:
We are setting that prop in SimpleSelect line 683. 'aria-multiline': 'false' and role: 'textbox' are not required to work with JAWS but are recommended properties in the structure.

We can eventually get rid of them.